Data/Tools/ezEditor/Localization/en/ScenePlugin.txt

```diff
@@ -405,4 +405,5 @@ ezClothSheetFlags::FixedEdgeRight;Edge: Right
 ezClothSheetFlags::FixedEdgeBottom;Edge: Bottom
 ezClothSheetFlags::FixedEdgeLeft;Edge: Left
 ezClothSheetFlags::FixedEdgeLeft;Edge: Left
-ezCustomMeshComponent;Custom Mesh Component
\ No newline at end of file
+ezBoxReflectionProbeComponent;Box Reflection Probe
+ezSphereReflectionProbeComponent;Sphere Reflection ProbeezCustomMeshComponent;Custom Mesh Component
```

Comment:
something got mixed up here

Reply:
btw, you can insert a link to your new component documentation here

Reply:
Done!